Patch against rev 98 of https://svn.java.net/svn/svgsalamander~svn/trunk. Adds getXForm() method needed for the importvec plugin.
Index: core/src/com/kitfox/svg/TransformableElement.java
===================================================================
--- core.orig/src/com/kitfox/svg/TransformableElement.java	2012-05-11 17:03:52.404311564 +0200
+++ core/src/com/kitfox/svg/TransformableElement.java	2012-06-18 00:00:20.444550308 +0200
@@ -53,11 +53,6 @@
     {
         super(id, parent);
     }
-
-    public AffineTransform getXForm()
-    {
-        return xform;
-    }
 /*
     public void loaderStartElement(SVGLoaderHelper helper, Attributes attrs, SVGElement parent)
     {
